crypto/fipsmodule/evp/p_rsa.c

```diff
@@ -734,12 +732,12 @@ static int pkey_rsa_ctrl_str(EVP_PKEY_CTX *ctx, const char *type,
     // "max" and "auto" are not supported
     long saltlen;
 
-    if (!strcmp(value, "digest")) {
+    if (!strcmp(value, "digest") || !strcmp(value, "-1")) {
```

Comment:
This will always be true, unless `value` can magically compare positively to both "digest" and "-1" simultaneously.